File: Codigos em C/Lista 07-19/3. Chapa Quente.c
 #include <stdio.h>

int main() {
    
int Codigo,quantidade;
float preco;

printf ("Seja bem vindo a lanchonete\n");
printf ("                               CHAPA QUENTE \n");
printf ("\n");
printf ("            Cardapio: \n");
printf ("(200) Baguncinha        = 21.80 R$ \n");
printf ("(201) Bagunca caseiro   = 28.90 R$ \n");
printf ("(202) X salada          = 18.20 R$ \n");
printf ("(203) Bauru simples     = 15.90 R$ \n");
printf ("(204) Refrigerante lata = 06.00 R$ \n");
printf ("(205) Agua mineral      = 04.00 R$ \n");


    printf("Digite o Codigo referente ao lanche: ");
    scanf("%i", &Codigo);
    printf("Digite a quantidade:");
    scanf ("%i", &quantidade);
    
switch (Codigo) {
    case 200:
    printf ("voce escolheu %i Baguncinha\n", quantidade);
    preco = quantidade * 21.80;
    printf ("O valor total é %.2f R$", preco);
    break;
    case 201:
        printf ("voce escolheu %i Bagunca caseiro\n", quantidade);
    preco = quantidade * 28.90;
    printf ("O valor total é %.2f R$", preco);
    break;
    case 202:
        printf ("voce escolheu %i X salada\n", quantidade);
    preco = quantidade * 18.20;
    printf ("O valor total é %.2f R$", preco);
    break;
    case 203:
        printf ("voce escolheu %i Bauru simples\n", quantidade);
    preco = quantidade * 15.90;
    printf ("O valor total é %.2f R$", preco);
    break;
    case 204:
        printf ("voce escolheu %i Refrigerante lata\n", quantidade);
    preco = quantidade * 6;
    printf ("O valor total é %.2f R$", preco);
    break;
    case 205:
        printf ("voce escolheu %i Agua mineral\n", quantidade);
    preco = quantidade * 4;
    printf ("O valor total é %.2f R$", preco);
    break;
default:
  printf("A letra digitada é uma consoante.\n");
 break;
    }
printf("\n");
    return 0;
}

File: Codigos em Python/Lista 07-19/3. Chapa Quente.py
print("Seja bem vindo a lanchonete")
print("                               CHAPA QUENTE ")
print("\n")
print("            Cardapio: ")
print("(200) Baguncinha        = 21.80 R$")
print("(201) Bagunca caseiro   = 28.90 R$")
print("(202) X salada          = 18.20 R$")
print("(203) Bauru simples     = 15.90 R$")
print("(204) Refrigerante lata = 06.00 R$")
print("(205) Agua mineral      = 04.00 R$")

Codigo = int(input("Digite o Codigo referente ao lanche: "))
quantidade = int(input("Digite a quantidade: "))

preco = 0

if Codigo == 200:
    print(f"Você escolheu {quantidade} Baguncinha")
    preco = quantidade * 21.80
    print(f"O valor total é {preco:.2f} R$")
elif Codigo == 201:
    print(f"Você escolheu {quantidade} Bagunca caseiro")
    preco = quantidade * 28.90
    print(f"O valor total é {preco:.2f} R$")
elif Codigo == 202:
    print(f"Você escolheu {quantidade} X salada")
    preco = quantidade * 18.20
    print(f"O valor total é {preco:.2f} R$")
elif Codigo == 203:
    print(f"Você escolheu {quantidade} Bauru simples")
    preco = quantidade * 15.90
    print(f"O valor total é {preco:.2f} R$")
elif Codigo == 204:
    print(f"Você escolheu {quantidade} Refrigerante lata")
    preco = quantidade * 6.00
    print(f"O valor total é {preco:.2f} R$")
elif Codigo == 205:
    print(f"Você escolheu {quantidade} Agua mineral")
    preco = quantidade * 4.00
    print(f"O valor total é {preco:.2f} R$")
else:
    print("O codigo digitado não é valido")
